recipe/meta.yaml

```
@@ -12,63 +12,39 @@ source:
     - fix-doc-path.patch  # [build_platform != target_platform]
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - {{ cdt('mesa-dri-drivers') }}  # [linux]
-    - {{ cdt('libselinux') }}  # [linux]
-    - {{ cdt('libxdamage') }}  # [linux]
-    - {{ cdt('libxfixes') }}  # [linux]
-    - {{ cdt('libxxf86vm') }}  # [linux]
-    - {{ cdt('expat') }}  # [linux]
-    - {{ cdt('libxext') }}  # [linux]
-    - {{ cdt('libxrender') }}  # [linux]
-    - {{ cdt('libxt') }}  # [linux]
-    - {{ cdt('libx11-devel') }}  # [linux]
-    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
-    - libxcb  # [linux]
+
     - pkg-config
     - gnuconfig  # [unix]
     - make  # [unix]
     - zlib  # [build_platform != target_platform]
+    - libiconv  # [build_platform != target_platform and linux]
```

now maybe just check one last time that you need these last two
```suggestion
    # - zlib  # [build_platform != target_platform]
    # - libiconv  # [build_platform != target_platform and linux]
```

---

zlib was a good call - libiconv I bet is needed tho. I needed to hack the linking arg in here in order to make it work: https://github.com/conda-forge/gnuplot-feedstock/pull/61/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bcR45